EIPS/eip-compliant.md

```diff
@@ -0,0 +1,198 @@
+---
+eip: <to be assigned>
+title: Compliant Token
+author: Daniel Lehrner <daniel@io.builders>
+discussions-to: https://github.com/IoBuilders/EIPs/pull/2
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-04-29
+requires: 1462
+---
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+
+An extension to the ERC-1462 token which adds checking functions for holdable, clearable, fundable and payoutable tokens.
```

Comment:
I suggest to add more context, referencing the payment market and emoney scenario, referencing kyc limits